wolfcrypt/benchmark/benchmark.c

```diff
@@ -32,6 +32,13 @@
 /* Macro to disable benchmark */
 #ifndef NO_CRYPT_BENCHMARK
 
+#ifdef WOLFSSL_STATIC_MEMORY
+    #include <wolfssl/wolfcrypt/memory.h>
```

Comment:
There is an issue with unknown type after WOLFSSL_STATIC_MEMORY has been undefined in memory.h. "./configure --enable-staticmemory --enable-iopool"